src/lib/people/team/team.ts

```diff
@@ -0,0 +1,47 @@
+import * as v from 'valibot';
+
+export const committees = [
+    'Executive',
+    'Innovation',
+    'Service',
+    'External Relations',
+    'Membership & Internals',
+    'Branding & Creatives',
+    'Engineering',
+];
+
+const MemberSchema = v.object({
+    name: v.string(),
+    title: v.string(),
+    committee: v.pipe(
+        v.string(),
+        v.check(i => committees.includes(i), 'invalid committee'),
+    ),
+    socials: v.optional(v.record(v.string(), v.pipe(v.string(), v.url()))),
+});
+
+export interface Member extends v.InferOutput<typeof MemberSchema> {
+    img: string;
+}
+
+const baseRe = /(?<base>\w+).json$/;
+
+export async function getTeam(): Promise<Member[]> {
```

Comment:
We can actually omit the return type here because TypeScript is smart enough to infer this from the function's `return` value.
```suggestion
export async function getTeam() {
```